modules/developer_manual/pages/_partials/app/advanced/security/idp_setup/example_config.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$CONFIG = array (
```

Comment:
are these files referenced anywhere? THX

Reply:
I've removed this file, map_user_example.php, and node-oidc-provider-config.js.